scripts/core/editor3/components/Editor3Component.jsx

```diff
@@ -428,7 +428,7 @@ export class Editor3Component extends React.Component {
 
                             const selectionRect = getVisibleSelectionRect(window);
 
-                            if (selectionRect != null) {
+                            if (this.editorNode.dataset != null && selectionRect != null) {
```

Comment:
Better check if `this.editorNode != null`, because if it isn't, DOM node will always have `dataset`